### PR TITLE
Reuse graphjs components from the vuejs admin, fix #167

### LINKED ISSUE #167: Reuse graphjs components from the vuejs admin

### PR DESCRIPTION
**Pull request by davidbgk**
I tried to be consistent with both front and admin dashboards:

capture d ecran 2015-09-17 a 09 09 22

capture d ecran 2015-09-17 a 09 08 59

### CODE REVIEW

File: js/components/charts/widget.vue [hunk 1]
```diff
@@ -53,10 +41,10 @@
 <script>
 'use strict';
```
Comment (noirbizarre) [inline js/components/charts/widget.vue]: Can be removed as it's implicit

File: js/components/charts/widget.vue [hunk 2]
```diff
@@ -53,10 +41,10 @@
...
-var $ = require('jquery'),
-    moment = require('moment'),
-    debounce = require('debounce'),
-    Chart = require('chart.js');
+import $ from 'jquery';
+import moment from 'moment';
+import debounce from 'debounce';
+import Chart from 'chart.js';
 
 require('Chart.StackedBar.js');
```
Comment (noirbizarre) [inline js/components/charts/widget.vue]: import ?

File: js/components/charts/widget.vue [hunk 3]
```diff
@@ -103,20 +91,13 @@ var AREA_OPTIONS = {
...
             chart: null,
-            charttype: 'Area',
             canvasHeight: null,
-            height: '300px',
-            types: {
-                Area: 'area-chart',
-                Bar: 'bar-chart',
-                StackedBar: 'bar-chart',
-                Line: 'line-chart'
-            }
```
Comment (noirbizarre) [inline js/components/charts/widget.vue]: Why remove the type ?

File: js/graphs.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+Vue.use(require('plugins/util'));
+Vue.use(require('plugins/text'));
+Vue.use(require('plugins/i18next'));
+Vue.use(require('plugins/markdown'));
+
+$(require('api')).on('built', function() {
+    new Vue(require('components/dashboard/graphs.vue'));
+});
+
```
Comment (noirbizarre) [inline js/graphs.js]: I think it should be mutualized with activities.js. COmmonChunk will be far less powerfull than a single JS.

File: udata/templates/site/dashboard.html [hunk 5]
```diff
@@ -10,8 +10,13 @@
 {% set body_class = 'dashboard' %}
 
 {% block extra_js %}
-<script type="text/javascript" src="{{ static('dashboard/site.js') }}"></script>
-<script type="text/javascript" src="{{ static('activities.js') }}"></script>
+<script src="{{ static('vue-common.js') }}"></script>
+<script src="{{ static('graphs.js') }}"></script>
+<script src="{{ static('activities.js') }}"></script>
```
Comment (noirbizarre) [inline udata/templates/site/dashboard.html]: graphs and activities can be merged, no ?

**Comment by noirbizarre** [comment]
Interesting change. Grouping the chart together give more spaces but brings scaling problem.
I think there should be an associated PR in gouvfr in order to remove legacy metrics theming and maybe theme this one.

File: js/components/charts/widget.vue (hunk 3 above)
Comment (davidbgk) [inline js/components/charts/widget.vue]: `types` are only useful to display the box to swap the chart types, I restored the different methods to build graphs though.

**Comment by davidbgk** [comment]
And for a given org:

capture d ecran 2015-09-17 a 16 16 32

**Pull request merged**